apps/api/v1/pages/api/selected-calendars/_post.ts

```diff
@@ -52,7 +53,9 @@ import {
 async function postHandler(req: NextApiRequest) {
   const { userId, isSystemWideAdmin } = req;
   const { userId: bodyUserId, ...body } = schemaSelectedCalendarBodyParams.parse(req.body);
-  const args: Prisma.SelectedCalendarCreateArgs = { data: { ...body, userId } };
+  const args = {
+    data: { ...body, userId } as Prisma.SelectedCalendarUncheckedCreateInput,
```

Comment:
Why casting instead of typing here?

Reply:
No specific reason. I guess autocomplete.

Fixed it.